core/nuxeo-operation.js

```diff
@@ -303,6 +304,12 @@ import './nuxeo-connection.js';
       return this._isView(input) && input.selectAllActive;
     }
 
+    _hasExcludedItems(input) {
+      return input.selectAllActive && input._excludedItems && input._excludedItems.length > 0
```

Comment:
suggestion code
```
const {selectAllActive,_excludedItems}=input
return selectAllActive && _excludedItems.length>0?_excludedItems:[]
```

why we are checking two times?
`input._excludedItems && input._excludedItems.length > 0`